src/Microsoft.DocAsCode.Metadata.ManagedReference/Parsers/TripleSlashCommentModel.cs

```diff
@@ -397,12 +395,8 @@ private string GetXmlValue(XPathNavigator node)
             // e.g.
             // <remarks><para>Value</para></remarks>
             // decode InnerXml as it encodes
-            var lineInfo = node as IXmlLineInfo;
-            var lineNumber = lineInfo.LineNumber - 1;
-            var line = _lines[lineNumber];
-            int column = GetNonWhitespaceIndex(line);
-
             var content = WebUtility.HtmlDecode(node.InnerXml);
+            int column = GetNonWhitespaceIndex(content.Split('\n')[0]);
```

Comment:
content is not accurate, it re-formats xml. The fix fails UT, https://ci.appveyor.com/project/vicancy/docfx/build/1.0.297